src/sprint_four_player/Headquarters.java

```diff
@@ -0,0 +1,132 @@
+package sprint_four_player;
+
+import battlecode.common.*;
+import sprint_three_player.Communication;
```

Comment:
This should be sprint_four_player. However, this import statement is not needed at all. I would say remove it. But, if it is kept, change it to sprint_four_player.

Reply:
Done! removed the requested change.